vulnerabilities/importers/github.py

```diff
@@ -451,6 +385,7 @@ def get_inferences(self, advisory_data: AdvisoryData) -> Iterable[Inference]:
             aff_vers, unaff_vers = resolve_version_range(
                 affected_version_range=affected_version_range,
                 package_versions=valid_versions,
+                ignorable_versions=WEIRD_IGNORABLE_VERSIONS,
```

Comment:
The more I see these ignorable_versions and in particular the list of "WEIRD_IGNORABLE_VERSIONS,"  the more I think this is smelling bad or weird. IMHO we should rather have a new  approach outlined in https://github.com/nexB/vulnerablecode/issues/737